frontend/package.json

```
@@ -56,6 +58,7 @@
     "lint:fix": "npm run lint -- --fix",
     "postinstall": "nuxt prepare",
     "preview": "nuxt preview",
+    "test": "npx playwright test --config=tests/playwright.config.ts",
```

we will add more `tests` like `unit` and `componets` soon, so this might describe your `tests`
 better```suggestion
    "teste2e": "playwright test --config=tests/playwright.config.ts",
```

---

good note, updated to "test:e2e": "playwright test --config=tests/playwright.config.ts",